A. Team/Program.cs

```diff
@@ -0,0 +1,52 @@
+﻿namespace A._Team
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            Reader rd = new Reader();
```

Comment:
Variables must have meaningful names
```suggestion
            var reader = new Reader();
```